src/docker-osx-dev

```diff
@@ -399,7 +399,7 @@ function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
   DOCKER_HOST_USER=$(inspect_docker_machine "{{.Driver.SSHUser}}")
   DOCKER_HOST_IP=$(inspect_docker_machine "{{.Driver.IPAddress}}")
-  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.StorePath}}")
+  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.HostOptions.AuthOptions.StorePath}}")
```

Comment:
I don't know. I definitely used older versions of `docker-osx-dev` with docker-machine and they worked so I suspect there's been a change.

What I don't know is if the fix here will work for older versions of docker-machine (I've upgraded). I think the `start` command was changed to error if the machine was already running, hence the problem?